lesson13_2.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+import json
+import logging
+
+path = Path('/Users/o.bogachenko/PycharmProjects/aqa4/Data_for_test/')
```

Comment:
this path will work only on your machine, but anyone should be able to run the code